dali/test/python/test_operator_gaussian_blur.py

```diff
@@ -81,28 +81,33 @@ def gaussian_baseline(image, sigma, window_size, axes=2, skip_axes=0, dtype=np.u
         return dtype(image + 0.5)
 
 
-def check_gaussian_blur(batch_size, sigma, window_size, op_type="cpu"):
-    decoder_device = "cpu" if op_type == "cpu" else "mixed"
+def get_gaussian_pipe(batch_size, sigma, window_size, op_type):
     pipe = Pipeline(batch_size=batch_size, num_threads=4, device_id=0)
     with pipe:
         input, _ = fn.file_reader(file_root=images_dir, shard_id=0, num_shards=1)
-        decoded = fn.image_decoder(input, device=decoder_device, output_type=types.RGB)
+        decoded = fn.image_decoder(input, device="cpu", output_type=types.RGB)
+        if op_type == "gpu":
+            decoded = decoded.gpu()
```

Comment:
It is not needed I think. It is a hidden future of the functional API. @mzient right?

Reply:
Will check, force of habit.

Reply:
So this is actually needed, it works the other way round, so no need for `device="gpu"` here:
`fn.gaussian_blur(decoded.gpu())`